src/protocols/raydium.ts

```diff
@@ -28,6 +28,7 @@ import {
 } from "../ids";
 import { Gateway } from "@dappio-wonderland/gateway-idls";
 import { PoolInfoWrapper } from "@dappio-wonderland/navigator/dist/raydium";
+import { getMultipleAccounts } from "@dappio-wonderland/navigator/dist/utils";
```

Comment:
This works for now. However imo `getMultipleAccounts` should be exported in `index.ts` from Navigator as well

Reply:
Try `utils. getMultipleAccounts`

Reply:
Sure!